server/utils/sendBill.js

```diff
@@ -0,0 +1,71 @@
+const nodemailer = require('nodemailer');
+
+const sendTheBill = async (
+  to,
+  subject,
+  { total, hourPrice, hourNumber, description, client, provider },
+  next
+) => {
+  try {
+    const smtpTransport = nodemailer.createTransport({
+      service: 'Gmail',
+      auth: {
+        user: process.env.NODEMAILER_SENDER_EMAIL,
+        pass: process.env.SENDER_EMAIL_PASSWORD,
```

Comment:
I like that you hide your email information, for more improvement destruct the values from process.env
```js
    const { NODEMAILER_SENDER_EMAIL, SENDER_EMAIL_PASSWORD } = process.env;

```